ls1intum/Artemis
# [Feature] Fixed Date  Validation

### Checklist


### Motivation and Context

Before the fix, you have been able to set the AssesmentDueDate before the Due Date

closes #3505

### Description

After checking wheter or not the ReleaseDate is before the AssesmentDueDate, the condtion went into the else block which sett the flag on false (meaning there is no error) without checking the DueDate of the Excercise


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new text/modeling or file upload excercise.
4. Try to set the assesment due date between the realse and the due date
5. See the correct behaviour
6. Try steps 4 and 5 with an already existing excercise

### Test Coverage




### Screenshots



Old behaviour:
![image](https://user-images.githubusercontent.com/65229601/120023002-7b329900-bfed-11eb-9a33-76120f7a06f4.png)

New behaviour:
![image](https://user-images.githubusercontent.com/65229601/120023038-871e5b00-bfed-11eb-9b5e-aae87b26a19f.png)


This is my first pull request, so i am thankful for any tips and help
